cmd/docgenerate/main.go

```diff
@@ -0,0 +1,175 @@
+// Command docgenerate generates static documentation for Chain Core and the
+// Chain Core SDKs.
+//
+// Usage:
+//
+//     docgenerate chainPath outputDir
```

Comment:
I went back and forth on that myself. I think after a conversation we had earlier in the week about someone struggling with `build-cored-release`, it occurred to me that it was nice if a tool didn't have `$CHAIN` as a dependency if it didn't absolutely have to.

At any rate, the `upload-docs` script (which is probably how `docgenerate` is invoked 95% of the time) feeds `$CHAIN` as a parameter here, so I don't think we'll even have to worry about the extra verbosity in practice.